hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/HoodieTable.java

```diff
@@ -367,8 +367,8 @@ public HoodieTimeline getCompletedCommitTimeline() {
   /**
    * Get only the inflights (no-completed) commit timeline.
    */
-  public HoodieTimeline getPendingCommitTimeline() {
-    return metaClient.getCommitsTimeline().filterPendingExcludingMajorAndMinorCompaction();
+  public HoodieTimeline getPendingCommitsTimeline() {
+    return metaClient.getCommitsTimeline().filterInflightsAndRequested();
```

Comment:
Does the behavior change for  MOR table already because compaction and log_compaction got involved?

Reply:
The behavior will not change, because the Timeline Action returned by calling HoodieTableMetaClient#getCommitsTimeline does not include Compaction and LogCompaction.

so I change HoodieTimeline#filterPendingExcludingMajorAndMinorCompaction -> filterInflightsAndRequested